connect_sample/src/main/java/com/zendesk/sample/connect/SampleMessageReceiver.java

```diff
@@ -0,0 +1,45 @@
+package com.zendesk.sample.connect;
+
+import android.app.Service;
+
+import com.google.firebase.messaging.RemoteMessage;
+import com.zendesk.connect.MessageReceiver;
+import com.zendesk.logger.Logger;
+
+/**
+ * You can optionally extend {@link MessageReceiver} to react to Firebase events like message received
```

Comment:
```suggestion
 * You can optionally provide an implementation of {@link MessageReceiver} to react to Firebase events like message received
```

The same suggestion goes for `NotificationEventListener` and `NotificationFactory`

Reply:
Good call, I was just reading the feedback from Chuck in the docs PR. It makes sense to have this updated as well.